### DISCUSSION TITLE
2009/01/30/%E6%A2%A6%E4%B8%AD%E7%9A%84%E6%89%8D%E5%AD%90

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#250] [Announcements]
# 2009/01/30/%E6%A2%A6%E4%B8%AD%E7%9A%84%E6%89%8D%E5%AD%90

半夜的时候，老婆一翻身把我弄醒了。我当时正在做梦，并且在梦中创作了一首歌曲。我看老婆也醒了，就对她说：“我刚创作了一首歌曲，你记性好，快帮我记下来，否则我明早一准忘掉。”于是我就把刚创作的音乐哼哼了一遍给老婆，又自己哼哼了几遍，觉得非常优美，才心满意足地继续睡觉了。

https://blog.qizhen.xyz/2009/01/30/%E6%A2%A6%E4%B8%AD%E7%9A%84%E6%89%8D%E5%AD%90

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Shaodong
2009年04月9日 15:00 编辑
看来真要摆个录音机或者录像机之类的~

回复
帆
2009年02月25日 15:52 编辑
我最搞的一个梦是梦见全校MM都长成一个样子了，惊魂甫定地回寝室。赶紧汇报。寝室一个预备党员悲痛欲绝：这样没有美女的惊喜，以后去自习室有什么乐趣啊？另一个同学很阴阳怪气说：这下可好了，以后找GF不用攀比了
